temporalcli/commandsmd/commands.md

```diff
@@ -1,221 +1,537 @@
-      * `* has-init` - Will assume an `initCommand` method is on the command
-      * `* exact-args=<number>` - Require this exact number of args
-      * `* maximum-args=<number>` - Require this maximum number of args
-  * Can have `#### Options` or `#### Options set for <options-set-name>` which can have options.
-    * Can have bullets
-      * Each bullet is `* <option-names> (<data-type>) - <short-description>. <extra-attributes>`.
-      * `<option-names>` is `` `--<option-name>` `` and can optionally be followed by ``, `-<short-name>` ``.
-      * `<data-type>` must be one of `bool`, `duration`, `int`, `string`, `string[]`, `string-enum`, `timestamp`, TODO: more
-      * `<short-description>` can be just about anything so long as it doesn't match trailing attributes. Any wrap
-        around to newlines + two-space indention is trimmed to a single space.
-      * `<extra-attributes>` can be:
-        * `Required.` - Marks the option as required.
-        * `Default: <default-value>.` - Sets the default value of the option. No default means zero value of the type.
-        * `Options: <option>, <option>.` - Sets the possible options for a string enum type.
-        * `Env: <env-var>.` - Binds the environment variable to this flag.
-        * ``Alias: `--<alias>`.`` - Sets a flag as an alias of this one. The flag should not be separately defined.
-      * Options should be in order of most commonly used.
-    * Also can have single lines below options that say
-      `Includes options set for [<options-set-name>](#options-set-for-<options-set-link-name>).` which is the equivalent
-      of just having them appended to the bulleted list.
-      * Just because a command may have a couple of similar options with another doesn't mean you _have_ to make a
-        shareable options set. Copy/paste is acceptable.
-* Keep commands in alphabetical order.
-* Commands that have subcommands cannot be run on their own.
-* Keep lines at 120 chars if possible.
-* Add punctuation even at the end of phrases.
+This document is automatically parsed.
+Follow these rules.
+
+IN-HOUSE STYLE
+
+* Wording and grammar.
+  * Run a spell check.
+  * Be clear and concise.
+  * Don't reword the command in the short description. Use distinct supplementary language.
+    * Yes: temporal workflow delete: Remove Workflow Execution
+    * No: temporal workflow delete: Delete Workflow
+  * Re-use and adapt existing wording and phrases wherever possible.
+  * Word command short descriptions as if they began "This command will..."
+    Use sentence casing for the short description.
+  * ID is fully capitalized in text ("the Workflow ID") and Id in [metasyntax](https://en.wikipedia.org/wiki/Metasyntactic_variable) (YourWorkflowId).
+* Avoid parentheticals unless absolutely necessary.
+
+* Wrapping:
+  * Assume a user-visible line length of 80 characters max. `fmt -w 78` can help.
```

Comment:
I don't think this is reasonable nor do I think this document needs to be formatted as if it's user-visible. The result of the markdown may be user-visible but not this document. I think the 120 char max that was there is good enough. Why change? I don't think this document (that is parsed for code generation) needs to be confused with documentation repo markdown.

Reply:
I have made a hard decision on the 80-column rule, because it is visible to users in the outputted documentation, and most users' terminals default to 80 columns.

Reply:
Right, I think that's a reasonable rendering rule but not usually a writing one. Having said that, this will change when we go to YAML (where there the rendering does match the writing), so no need to change anything now.

Reply:
Thank you for checking in on this @cretz. Glad to see that the 80-column rule is preserved for base UX.